specs/waterOrderingSpec.js

```diff
@@ -0,0 +1,37 @@
+const assert = require('assert')
+const rewiremock = require('./helpers/rewiremock')
```

Comment:
@nikitasakov I need to access it before relative imports, but it will be contrary to the above conditions (absolute then relative).
So, we have 3 options here. 
1. Keep it as it is.
2. Move rewiremock to the `index.js` but we need to place `require('./helpers')` in top of imports:
```js
const assert = require('assert')
const { getBasicController, rewiremock } = require('./helpers') // LOOK HERE
rewiremock('@sendgrid/mail').with(sendgridMock())
rewiremock('https').callThrough().with(httpsGetMock())
rewiremock.enable()
const waterOrderingController = require('../features/waterOrdering')
rewiremock.disable()
const replies = require('../features/waterOrdering/replies.js')
```
3. Require rewiremock in the `water_ordering_spec.js`:
```js
const assert = require('assert')
const rewiremock = require('rewiremock/node');  // LOOK HERE
rewiremock('@sendgrid/mail').with(sendgridMock())
rewiremock('https').callThrough().with(httpsGetMock())
rewiremock.enable()
const waterOrderingController = require('../features/waterOrdering')
rewiremock.disable()
const replies = require('../features/waterOrdering/replies.js')
const { getBasicController } = require('./helpers')  // LOOK HERE
```
but additional configuration may be necessary in the future and it is better to use a separate file for it.
Which method do you prefer?

Reply:
@npupko Can't it be like this? I see only one absolute import and it is in the top. Or I missed something?
```
const assert = require('assert')

const { getBasicController, rewiremock } = require('./helpers')
const replies = require('../features/waterOrdering/replies.js')

rewiremock('@sendgrid/mail').with(sendgridMock())
rewiremock('https').callThrough().with(httpsGetMock())

rewiremock.enable()
const waterOrderingController = require('../features/waterOrdering')
rewiremock.disable()

```

Reply:
@nikitasakov Yes, will do it. 

I was meant `rewiremock('@sendgrid/mail')` can be basically treated as imports too, cos you can write it like this:

```js
// This 
rewiremock('@sendgrid/mail').with(sendgridMock())
// is equal to
rewiremock(() => require('@sendgrid/mail')).with(sendgridMock())
```